src/ast/ConstructorDecl.ts

```diff
@@ -14,8 +14,15 @@ export default class ConstructorDecl extends AstNode {
     params: VarList;
 
     public parse(context: Tokenizer): any {
+        let indentLevel: number = context.getCurrentLineTabLevel();
+        context.getAndCheckNext("constructor");
+        this.modifier = context.getNext();
+
+        if (context.getCurrentLineTabLevel() <= indentLevel) return;
+
+        context.getAndCheckNext("params");
         this.params = new VarList();
-        // TODO: implement the rest.
+        this.params.parse(context);
     }
```

Comment:
Are we going to allow the user to specify a return value for constructors?
I think not allowing that syntax makes sense since constructors always return whatever object its the constructor of- but want to just double check that we are considering it an error if the user adds a return value